### PR TITLE
Add an example fuzzer with AFL-Style UI

### PR DESCRIPTION
**Pull request by ToSeven**

### CODE REVIEW

**Comment by ToSeven** [comment]
Due to a lack of stages/stats.rs (Because https://github.com/AFLplusplus/LibAFL/pull/1351 is not merged into the main branch), This commit failed to build.

**Comment by tokatoka** [comment]
you can merge #1351 to this

**Comment by tokatoka** [comment]
can you cargo fmt?

**Comment by ToSeven** [comment]
> can you cargo fmt?

I found "cargo fmt" that is executed directory in root directory dosen't make ./fuzzer formatted : (

**Comment by tokatoka** [comment]
> I found "cargo fmt" that is executed directory in root directory dosen't make ./fuzzer formatted : (

yes you need to cargo fmt in each fuzzer

can you edit Makefile.toml for your fuzzer and pass ci? apparently parsing stdout doesn't work out for your fuzzer.

**Comment by domenukk** [comment]
there is also `./scripts/fmt_all.sh` that does all of the formatting

**Comment by domenukk** [comment]
There's still an error in the example fuzzer somewhere:
```
Fuzzer does not generate any testcases or any crashes
```

File: fuzzers/libfuzzer_libpng_AFLStyle_UI/Makefile.toml [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+windows_alias = "unsupported"
+
+[tasks.test_unix]
+script_runner = "@shell"
+script='''
+rm -rf libafl_unix_shmem_server || true
+(timeout 11s ./${FUZZER_NAME} >fuzz_stdout.log 2>/dev/null || true) &
+sleep 0.2
+timeout 10s ./${FUZZER_NAME} >/dev/null 2>/dev/null || true
+if [ -z "$(grep "corpus: 30" fuzz_stdout.log)" ]; then
```
Comment (tokatoka) [inline fuzzers/libfuzzer_libpng_AFLStyle_UI/Makefile.toml:165]: please change this.
you can, for example, use the corpus that saves into the disk and count the number of entries in the corpus folder

Comment (ToSeven) [reply fuzzers/libfuzzer_libpng_AFLStyle_UI/Makefile.toml:165]: yeah, I already know the root cause.

**Comment by domenukk** [comment]
I don't really see any output in the UI :D 
but all in all fuzzing seems to do something which is good, thanks

**Pull request merged**